### ISSUE TITLE
Guidance for search parametrs 

### ISSUE THREAD

**Issue by espinoj** [cmu-phil/tetrad#870]
e.g.: 

alpha values, penalties, etc.
	Rules of thumb that we have?
	Luis has a method of choosing the alpha value based on sample size
	First pass is to improve documentation about recommended values
	Different prior’s on graph structures i.e., hubs and power law

**Comment by jdramsey** [comment]
Regarding this, I will implement Luis's rule (easy) and see how it works, in, say, PC with Fisher Z.
-Joe

**Comment by jdramsey** [comment]
Oops, forgot about this. I'll put it back on my list.

**Comment by jdramsey** [comment]
By the way, in case it's gotten lost in the the shuffle, this is the paper.

[Sellke-Bayarri-Berger-calibration-of-P-2001.pdf](https://github.com/cmu-phil/tetrad/files/3559237/Sellke-Bayarri-Berger-calibration-of-P-2001.pdf)

**Comment by jdramsey** [comment]
OK, I believe I've done this right. I have added the formula (3) from the above paper to the Fisher Z test. It's optional; I added a parameter ("Yes if the Sellke et al. adjustment should be used for alpha values.") which when set to Yes applies this transform.

The adjustment is alpha = 1.0 / (1.0 + (1.0 / (-e * p * log(p)))). I believe e is the original alpha, p the p-value, and alpha the adjusted alpha.

There is one corner case that I know of--if p = 0, log(p) will be undefined. But in that case, alpha should be 1.

This does seem to help.

If this is good, it should be added to other independence tests that use p-values.
